Topic6/src/main/java/com/globant/app/ShoppingCartController.java

```diff
@@ -0,0 +1,100 @@
+package com.globant.app;
+
+import java.util.Collection;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.globant.model.Product;
+import com.globant.model.ShoppingCart;
+import com.globant.services.ShoppingCartService;
+import com.globant.services.ShoppingCartServiceImpl;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@RestController
+@RequestMapping("/carts")
+public class ShoppingCartController {
+
+	private ShoppingCartService shoppingCartService = new ShoppingCartServiceImpl();

```

Comment:
This should be using spring boot dependency injection, why instantiate a service manually?